src/Microsoft.Azure.WebJobs.Extensions.Kafka/Listeners/KafkaListener.cs

```diff
@@ -55,7 +63,8 @@ internal class KafkaListener<TKey, TValue> : IListener
             KafkaListenerConfiguration kafkaListenerConfiguration,
             bool requiresKey,
             IDeserializer<TValue> valueDeserializer,
-            ILogger logger)
+            ILogger logger,
+            FunctionDescriptor functionDescriptor)
```

Comment:
Just take the functionId as a param - you don't need the whole descriptor do you?